src/main/java/discord4j/discordjson/json/OnboardingData.java

```diff
@@ -0,0 +1,29 @@
+package discord4j.discordjson.json;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import discord4j.discordjson.Id;
+import java.util.List;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableOnboardingData.class)
+@JsonDeserialize(as = ImmutableOnboardingData.class)
+public interface OnboardingData {
+
+    static ImmutableOnboardingData.Builder builder() {
+        return ImmutableOnboardingData.builder();
+    }
+
+    @JsonProperty("guild_id")
+    Id guildId();
+
+    List<OnboardingPromptData> prompts();
+
+    @JsonProperty("default_channel_ids")
+    List<Id> defaultChannelsIds();
```

Comment:
Let's use `defaultChannelIds` instead

Reply:
okay i based this in another fields with the same thing about the channel in singular and ids. but changing.